src/auto/border.rs

```diff
@@ -0,0 +1,34 @@
+// This file was generated by gir (8080733) from gir-files (469db10)
+// DO NOT EDIT
+
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct Border(Boxed<ffi::GtkBorder>);
+
+    match fn {
+        copy => |ptr| ffi::gtk_border_copy(mut_override(ptr)),
+        free => |ptr| ffi::gtk_border_free(ptr),
+        get_type => || ffi::gtk_border_get_type(),
+    }
+}
+
+impl Border {
```

Comment:
Unusable as no getters and no fields generated

Reply:
Apparently you apply things on the border but that's it. It's very strange so the current implementation seems complete.

Reply:
Bug in Gir/Gtk. I'll maybe file a bug. Don't know yet...

Reply:
Gir don't generated fields for records. This same as GdkRectange.